### PR TITLE
Adding a TIL on array duplication

### PR DESCRIPTION
**Pull request by msachi**
This is a common misunderstanding amongst beginners so I thought it would be useful to add it here.

### CODE REVIEW

File: javascript/duplicating-an-array.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+In JavaScript it is not possible to duplicate an array simply by defining a new variable and setting its value equal to the array.
```
Comment (alwinsol) [inline javascript/duplicating-an-array.md]: Good, I think its reads better like this

'It's not possible to duplicate an array, by simply defining a new variable and setting the value equal to the array in JavaScript.'

File: javascript/duplicating-an-array.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+This will only result in both the old and the new variable pointing to the same array. Any modifications to the 'new' array will also modify the original array.
+``` javascript
+var array1 = ['a','b','c'];
+var array2 = array1;
+array2.push('d');
+
+console.log(array1)   // ['a','b','c','d']
+```
+
+Instead, a neat way of copying an array is:
```
Comment (alwinsol) [inline javascript/duplicating-an-array.md]: For copying an array:

**Comment by msachi** [comment]
Thanks Alwin. Although I must admit I prefer my wording 🙈  As for the last comment, I believe the grammatically correct preposition is 'a good way _of_ doing something'. Hope that's ok : )

**Comment by alwinsol** [comment]
OK, its all semantics and personal preference.  Go with a good way and then lets merge :-)

**Comment by msachi** [comment]
Thanks : ) Have made some edits, mainly to the code. Let me know if it that's clear!

**Pull request merged**